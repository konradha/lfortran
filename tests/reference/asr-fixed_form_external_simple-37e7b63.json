{
    "basename": "asr-fixed_form_external_simple-37e7b63",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external_simple.f",
    "infile_hash": "cfe5f69b6617c6d433a46aa5a92950367a628b656e3311286ed665ae",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-fixed_form_external_simple-37e7b63.stderr",
    "stderr_hash": "13b4d060f4d5e8fcb0edc2e9821fe1602607735b82bf8ad1d9b639f6",
    "returncode": 1
}