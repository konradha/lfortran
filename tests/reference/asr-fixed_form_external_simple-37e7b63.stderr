Internal Compiler Error: Unhandled exception
Traceback (most recent call last):
[2m  Binary file "[0m[1m[35m$DIR/src/bin/lfortran[39m[0m[2m", in _start()[0m
[2m  File "[0m[1m[35m./csu/../csu/libc-start.c[39m[0m[2m", line 392, in __libc_start_main_impl()[0m
[2m  File "[0m[1m[35m./csu/../sysdeps/nptl/libc_start_call_main.h[39m[0m[2m", line 58, in __libc_start_call_main()[0m
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 1674, in ??[0m
    return emit_asr(arg_file, lfortran_pass_manager,
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 552, in ??[0m
    r = fe.get_asr2(input, lm, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 226, in LFortran::FortranEvaluator::get_asr2(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, LFortran::LocationManager&, LFortran::diag::Diagnostics&)[0m
    Result<ASR::TranslationUnit_t*> res2 = get_asr3(*ast, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 248, in LFortran::FortranEvaluator::get_asr3(LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&)[0m
    compiler_options.symtab_only, compiler_options);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_to_asr.cpp[39m[0m[2m", line 65, in LFortran::ast_to_asr(Allocator&, LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&, LFortran::SymbolTable*, bool, LFortran::CompilerOptions&)[0m
    LFORTRAN_ASSERT(asr_verify(*tu));
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_verify.cpp[39m[0m[2m", line 521, in LFortran::asr_verify(LFortran::ASR::TranslationUnit_t const&, bool)[0m
    v.visit_TranslationUnit(unit);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_verify.cpp[39m[0m[2m", line 111, in LFortran::ASR::VerifyVisitor::visit_TranslationUnit(LFortran::ASR::TranslationUnit_t const&)[0m
    this->visit_symbol(*a.second);
[2m  File "[0m[1m[35m$DIR/src/libasr/../libasr/asr.h[39m[0m[2m", line 3821, in LFortran::ASR::BaseVisitor<LFortran::ASR::VerifyVisitor>::visit_symbol(LFortran::ASR::symbol_t const&)[0m
    void visit_symbol(const symbol_t &b) { visit_symbol_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/../libasr/asr.h[39m[0m[2m", line 3595, in ??[0m
    case symbolType::Function: { v.visit_Function((const Function_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_verify.cpp[39m[0m[2m", line 251, in LFortran::ASR::VerifyVisitor::visit_Function(LFortran::ASR::Function_t const&)[0m
    visit_expr(*x.m_args[i]);
[2m  File "[0m[1m[35m$DIR/src/libasr/../libasr/asr.h[39m[0m[2m", line 3877, in LFortran::ASR::BaseVisitor<LFortran::ASR::VerifyVisitor>::visit_expr(LFortran::ASR::expr_t const&)[0m
    void visit_expr(const expr_t &b) { visit_expr_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/../libasr/asr.h[39m[0m[2m", line 3704, in ??[0m
    case exprType::Var: { v.visit_Var((const Var_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_verify.cpp[39m[0m[2m", line 337, in LFortran::ASR::VerifyVisitor::visit_Var(LFortran::ASR::Var_t const&)[0m
    require(symtab_in_scope(current_symtab, x.m_v),
LCompilersException: ASR verify failed: Var::m_v `b` cannot point outside of its symbol table
